src/app/events/components/test-edit-grades/test-edit-grades.component.html

```diff
@@ -145,7 +145,7 @@
               <span>{{ studentGrade.student.FullName }}</span>
               <span class="mobile mean">
                 {{ "tests.mean" | translate }}:
-                {{ studentGrade.finalGrade?.average | number: "1.0-3" }}
+                {{ studentGrade.finalGrade?.average | decimalOrDash: "1-3" }}
```

Comment:
Wäre es dann hier nicht "0-3"?

Reply:
Du meinst im Sinne von «wir behalten es gleich»? Ich habe das so angepasst, weil es IMO Sinn macht, dass es bei Dezimalzahlen immer ein Komma gibt (z.B. "5.0" und nicht "5"), würde ja in einem Zeugnis wohl auch so stehen. Ich frage aber zur Sicherheit noch einmal bei Franziska nach.

Reply:
Ist gemäss Franziska gut mit `1-3`.